.nvmrc

```diff
@@ -1 +1 @@
-lts/dubnium
+10
```

Comment:
This was switched to a number because `actions/setup-node` requires a number and can't use `lts/dubnium`. While I still like the signal that we use `lts` here, having a single source of truth for the node version we're on is nice.